stripper/ze_ancient_wrath_v2_test27_cm1.cfg

```diff
@@ -0,0 +1,759 @@
+:Fix no kevlar on new round
```

Comment:
I was originally doing individual comments on this file, but since many things come up multiple times I'll just do it all in one.

1) Comments are used very inconsistently, there seems to be many changes lumped under comments that are completely irrelevant to what the comment is saying

2) Formatting seems a bit off in places, e.g. multiple line breaks

3) Some of these nerfs seem very excessive, especially for a map we have never played before? Why are you sure these are necessary?